PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx

```diff
@@ -564,6 +694,14 @@ AliAnalysisTaskGammaConvV1::AliAnalysisTaskGammaConvV1(const char *name):
   DefineOutput(7, TTree::Class());
   DefineOutput(8, TTree::Class());
   DefineOutput(9, TTree::Class());
+  DefineOutput(10, TTree::Class());
+  DefineOutput(11, TTree::Class());
+  DefineOutput(12, TTree::Class());
+  DefineOutput(13, TTree::Class());
+  DefineOutput(14, TTree::Class());
+  DefineOutput(15, TTree::Class());
+  DefineOutput(16, TTree::Class());
+  DefineOutput(17, TTree::Class());
```

Comment:
Is it possible to define this many output containers?  Why are so many needed?

Reply:
I will be using https://hipe4ml.github.io/ to analyze the data afterward. The package mentioned here to my knowledge can only access trees from a root file (and not from a directory/subdirectory) and hence the usage of containers to store the trees. The containers seem to be working locally at least for this purpose